pkgs/os-specific/linux/firmware/fwupd/install-fwupdplugin-to-out.patch

```diff
@@ -1,37 +1,48 @@
 diff --git a/libfwupdplugin/meson.build b/libfwupdplugin/meson.build
-index 1afa28e1..3da81d30 100644
+index 71667b6bc..b944d97f2 100644
 --- a/libfwupdplugin/meson.build
 +++ b/libfwupdplugin/meson.build
-@@ -220,7 +220,8 @@ fwupdplugin = library(
+@@ -250,7 +250,7 @@ fwupdplugin = library(
    ],
    link_args: cc.get_supported_link_arguments([vflag]),
    link_depends: fwupdplugin_mapfile,
--  install: true
-+  install: true,
-+  install_dir: bindir / '..' / 'lib',
+-  install_dir: libdir_pkg,
++  install_dir: bindir / '..' / 'lib' / f'fwupd-@fwupd_version@',
```

Comment:
Maybe we should create a variable `daemonlibdir = bindir / '..' / 'lib' / f'fwupd-@fwupd_version@'` in the top-level `meson.build` so that we can re-use it in `install_dir`s and `install_rpath`s.

Reply:
At this point it would be so much easier to just combine out and lib.